project/project/CakePlatformDependencies.scala

```diff
@@ -62,14 +62,14 @@ object CakePlatformDependencies {
   }
 
   object ApacheCommons {
-    val codec: ModuleID = "commons-codec" % "commons-codec" % "1.6"
-    val compress: ModuleID = "org.apache.commons" % "commons-compress" % "1.9"
-    val lang3: ModuleID = "org.apache.commons" % "commons-lang3" % "3.4"
-    val logging: ModuleID = "commons-logging" % "commons-logging" % "1.1.3"
+    val codec: ModuleID = "commons-codec" % "commons-codec" % "1.11"
+    val compress: ModuleID = "org.apache.commons" % "commons-compress" % "1.15"
+    val lang3: ModuleID = "org.apache.commons" % "commons-lang3" % "3.6"
+    val logging: ModuleID = "commons-logging" % "commons-logging" % "1.2"
   }
 
   object Gatling {
-    val version = "2.2.3"
+    val version = "2.3.0"
```

Comment:
Can't we delete all this gatling stuff? I'm suggesting this as sbt-cake is not using this code - IIRC, it should be delegated to templates and projects.

Reply:
In fact, I'm pretty sure we can make major cleanups here wrt these dependencies - most of them are just not needed by sbt-cake?

Reply:
I'd be happy to remove it :)

Reply:
It means we should get rid to the performance tests tasks. Is it OK for you ?

Reply:
I'd rather leave it unchanged now and see what we can do when we address https://github.com/cakesolutions/sbt-cake/issues/64